### PR TITLE
fix(main/luvi): fix incompatible bytecode error

### PR DESCRIPTION
**Pull request by mbekkomo**
```
$ luvi . --
init.lua: cannot load incompatible bytecode
stack traceback:
        [C]: at 0x61b25d8c94
        [C]: in function 'require'
        [string "return require('init')(...)"]:1: in main chunk
```

This PR also bumps the commit used because previously the commit it was referencing did not exist.

### CODE REVIEW

**Comment by mbekkomo** [comment]
From PR: #21722

File: packages/luvi/build.sh [hunk 1]
```diff
@@ -17,36 +17,29 @@ TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 	-DWithPCRE2=On
 	-DWithSharedPCRE2=On
 	-DWithLPEG=On
+	-DLIBLUV_INCLUDE_DIR=${TERMUX_PKG_SRCDIR}/deps/luv/src
+	-DLIBLUV_LIBRARIES=${TERMUX_PREFIX}/lib/libluv.so
```
Comment (Grimler91) [inline packages/luvi/build.sh:21]: Generally speaking the headers and the library should come from the same place/the same version. Why use `-DLIBLUV_INCLUDE_DIR=${TERMUX_PKG_SRCDIR}/deps/luv/src` and not `-DLIBLUV_INCLUDE_DIR=${TERMUX_PREFIX/include/luv`?

File: packages/luvi/build.sh [hunk 2]
```diff
@@ -17,36 +17,29 @@ TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
...
-		-DLIBLUV_LIBRARIES=$lib/libluv.so
-		-DLUAJIT_INCLUDE_DIR=$include/luajit-2.1
-		-DLUAJIT_LIBRARIES=$lib/libluajit.so
-		-DLIBUV_INCLUDE_DIR=$include
-		-DLIBUV_LIBRARIES=$lib/libuv.so
-		-DOPENSSL_INCLUDE_DIR=$include
-		-DOPENSSL_LIBRARIES=$lib
-		-DPCRE2_INCLUDE_DIR=$include
-		-DPCRE2_LIBRARIES=$lib
-	"
```
Comment (Grimler91) [inline packages/luvi/build.sh:39]: Seems like all the options are duplicated here, I don't think that should be necessary. Please correct me if I am missing something

**Review by Grimler91 — CHANGES_REQUESTED** [review]
Thanks for the fix! Please see comments/questions

File: packages/luvi/build.sh (hunk 1 above)
Comment (mbekkomo) [reply packages/luvi/build.sh:21]: I've tried to do that when trying to submit this package, it somewhat failed and I have no idea why. But I'll try to add it back.

File: packages/luvi/build.sh (hunk 2 above)
Comment (mbekkomo) [reply packages/luvi/build.sh:39]: I don't get the meaning of "duplicated" here, could you rephrase it?

Comment (Grimler91) [reply packages/luvi/build.sh:39]: Oh, sorry, I think I was fooled by my browser, I thought these lines were added but they are removed. Ignore this :)

File: packages/luvi/build.sh (hunk 1 above)
Comment (Grimler91) [reply packages/luvi/build.sh:21]: I think we risk issues if version that is stored in luvi's deps/luv/src/ deviates from version that is stored in $PREFIX/, would be great if we could use $TERMUX_PREFIX/include/luv instead, so please check what error it gives

**Comment by TomJo2000** [comment]
Welp I tried to fix the branch being messed up, and messed it up even more by forgetting to commit.

**Comment by TomJo2000** [comment]
I might be able to fix it by reopening the PR.

**Comment by Biswa96** [comment]
Superseded by #22053
